feature/git/dffml_feature_git/feature/operations.py

```diff
@@ -398,7 +400,7 @@ def git_repo_release_valid_version(tag):
     return {"authors": len(author_lines.keys())}
 
 
-@op(inputs={"repo": git_repository}, outputs={}, stage=Stage.CLEANUP)
-async def cleanup_git_repo(repo: Dict[str, str]):
-    shutil.rmtree(repo.directory)
+@op(inputs={"record": git_repository}, outputs={}, stage=Stage.CLEANUP)
```

Comment:
Nothing in this file needs to be changed (`feature/git/dffml_feature_git/feature/operations.py`) all instances of repos were talking about git repos